### ISSUE TITLE
Log-exporter: Support shipping RDS logs to an S3 bucket

### ISSUE THREAD

**Issue by spgreenberg** [cloud-gov/aws-broker#135]
In order to support shipping RDS logs to customers via S3, the log-exporter needs to support copying download archives to S3 buckets.

Depends on: https://github.com/cloud-gov/aws-broker/issues/134

## Acceptance Criteria

The log-exporter needs to:
- [ ] Be able to copy downloaded archives to a customer S3 bucket
- [ ] Ensure logs are copied the correct bucket (and only the correct bucket).

--- 

## Security considerations

None provided the log exporter lands archives in the correct buckets.

**Comment by markdboyd** [comment]
Superseded by

https://github.com/cloud-gov/aws-broker/issues/372
https://github.com/cloud-gov/private/issues/957
